### PR TITLE
Improve handling of ccdb / condition options

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

File: Framework/Core/src/WorkflowCustomizationHelpers.cxx [hunk 1]
```diff
@@ -41,13 +41,21 @@ namespace o2::framework
 
 std::vector<ConfigParamSpec> WorkflowCustomizationHelpers::requiredWorkflowOptions()
 {
+  auto defaultConditionBackend = std::getenv("DDS_SESSION_ID") ? "http://o2ccdb.internal" : "http://alice-ccdb.cern.ch";
```
Comment (shahor02) [inline Framework/Core/src/WorkflowCustomizationHelpers.cxx]: Should be `http://o2-ccdb.internal`.

**Comment by ktf** [comment]
@TimoWilken I think there is something wrong with the builders. build/O2/o2 has been hanging since yesterday.

**Comment by davidrohr** [comment]
@ktf : I just tried with this PR, but it fails with
```
[47345:internal-dpl-ccdb-backend]: [09:16:39][FATAL] error while setting up workflow in o2-cpv-reco-workflow: missing option: condition-backend (No such node (condition-backend))
```

**Comment by ktf** [comment]
I am looking into it.

**Pull request closed without merging**